cmd/buildah/mount.go

```diff
@@ -51,7 +51,7 @@ func mountCmd(c *cobra.Command, args []string, noTruncate bool) error {
 		// Differently, allow the mount if we are already in a userns, as the mount point will still
 		// be accessible once "buildah mount" exits.
 		if os.Getenv(startedInUserNS) != "" && store.GraphDriverName() != "vfs" {
-			return fmt.Errorf("cannot mount using driver %s in rootless mode", store.GraphDriverName())
+			return fmt.Errorf("cannot mount using driver %s in rootless mode. You need to run it in a `buildah unshare` session", store.GraphDriverName())
```

Comment:
No change necessary, but I'm hoping there's enough bread crumbs here for the user to figure out how to solve things.  However the message is pretty long now and I can't come up with a better one.

LGTM